packages/flutter_test/lib/src/binding.dart

```diff
@@ -106,6 +111,32 @@ mixin TestDefaultBinaryMessengerBinding on BindingBase, ServicesBinding {
   }
 }
 
+/// An accessibility announcement made by the Flutter framework.
+///
+/// This class is mostly used by the testing api to store the announcements
+/// in a user-friendly structure so that the code consumers can check the details
+/// of announcements in their tests.
```

Comment:
Somewhere in this dartdoc I'd note that the fields of this class correspond to parameters of the `SemanticsService.announce` method.

Reply:
Done!